dev/docs/platform_integration/lib/android.dart

```diff
@@ -1,2 +1,2 @@
-/// [Flutter platform integration APIs for Android.](https://docs.flutter.io/javadoc/)
+/// [Flutter platform integration APIs for Android.](https://api.flutter.dev/javadoc/)
```

Comment:
I don't believe the redirects handle this properly - these two URLs are not the same - the .dev one is redirecting to the root (and non-folder URLs on the new domain are 404!).